e2e/tests/framework/generateLocalStorageData.e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/**
+ * This test suite is dedicated to generating LocalStorage via Session Storage to be used
+ * in some visual test suites like controlledClock.visual.spec.js. This suite should run to completion
+ * and generate an artifact in ./e2e/test-data/<name>_storage.json . This will run
+ * on every commit to ensure that this object still loads into tests correctly and will retain the
+ * *.e2e.spec.js suffix.
+ *
+ * TODO: Provide additional validation of object properties as it grows.
+ */
+
+const { test, expect } = require('../../pluginFixtures.js');
+const { createDomainObjectWithDefaults, createExampleTelemetryObject } = require('../../appActions.js');
+
+const overlayPlotName = 'Overlay Plot with Telemetry Object';
+
+test.describe('Generate Visual Test Data @localStorage @generatedata', () => {
+    // test.use({
+    //     clockOptions: {
+    //         shouldAdvanceTime: false //Don't advance the clock
+    //     }
+    // });
+
+    test('Generate Overlay Plot with Telemetry Object', async ({ page, context }) => {
+
+        //Go to baseURL
+        await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
```

Comment:
Let's move this to a `beforeEach` and factor it out of the other tests. Also, we can just change this to `await page.goto('./', { waitUntil: 'networkIdle' });`

The `?hideTree=true` param doesn't actually seem to work (I think it's parsed and handled after the params for bounds are set, so it just gets overwritten)